Makefile

```diff
@@ -5,6 +5,9 @@ venv:
 	python3 -m venv venv
 	venv/bin/pip install -e .[docs,test]
 
+lint: venv
+	venv/bin/mypy hcloud
```

Comment:
I first tarted with a pre-commit hook, but changed because my IDE don't have access to the pre-commit venv. Having mypy installed allows the IDE to also run the checks.

Not sure hat is best here. I am tempted to also have it in pre-commit.

Reply:
Let's leave like this.

We will soon add pylint to the lint task, and pylint is required to be part of the same venv as the code.